styles/globals.css

```diff
@@ -1,11 +1,80 @@
+:root {
+  --c-pride-blue: #001a98;
+}
+
 html,
 body {
   padding: 0;
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans",
     "Droid Sans", "Helvetica Neue", sans-serif;
+  font-size: 1.15rem;
 }
 
 * {
   box-sizing: border-box;
 }
+
+a,
+a:visited {
+  color: var(--c-pride-blue);
+  text-decoration: none;
+}
+
+a:hover,
+a:focus {
+  color: #000;
+}
+
+a::after {
+  content: "";
+  margin-top: -0.18rem;
+  display: block;
+  height: 0.15rem;
+  width: 100%;
+  background: rgb(229 0 0);
+  background: linear-gradient(
```

Comment:
A little out of my CSS depth here, but: could one store these gradients in variables? Maybe we want to use them again later on, or even just to give them a descriptive name might be good? (`pride-gradient` or something?)

Reply:
It might be possible! I need to do a bit of research on this 😄

Reply:
It is possible! I stored both of these to css-variables - naming is again hard, I ended up with `pride-gradient-rainbow` and `pride-gradient-progress` (as the hover/focus gradient is from the progress pride flag), but if you come up with any better names, let me know, I'm happy to change them 😄